### DISCUSSION TITLE
orthogroups of expansions/contractions

### DISCUSSION THREAD

**Discussion by rsandean** [hahnlab/CAFE5#138] [Q&A] [answered]
Hi, 

I have recently used CAFE5 for an analysis for my orthofinder results, I was wondering if there is a specific while which shows which orthogroups were expanded and contracted. I understand that there's the Base_clade_results.txt but i wasnt sure which orthogroups were increased or decreased. Thank you! :)

**Comment by benfulton** [comment, marked as answer]
The base_change.tab file has that information. It is a tab-delimited table, with each row representing a family and each column representing a location on the tree. The value shows how much the family increased or decreased at that location on the tree.

Let me know if you have any questions.

**Reply by Xueliang24** [reply]
Does it mean that the script `cafetutorial_report_analysis.py` could not work for *.cafe result files
